### ISSUE TITLE
OutputEmitter does not support for subscribe in v17.2.4

### ISSUE THREAD

**Issue by aiuoe** [angular/angular#54802]
### Which @angular/* package(s) are the source of the bug?

core

### Is this a regression?

No

### Description

/**
 * An `OutputEmitter` is created by the `output()` function and can be
 * used to emit values to consumers of your directive or component.
 *
 * Consumers of your directive/component can bind to the output and
 * subscribe to changes via the bound event syntax. For example:
 *
 * ```html
 * <my-comp (valueChange)="processNewValue($event)" />
 * ```
 *
 * @developerPreview
 */
export declare interface ɵOutputEmitter<T> {
    emit(value: T): void;
}

### Please provide the exception or error you saw

```true
Property 'subscribe' does not exist on type 'ɵOutputEmitter<void>'.
```

### Please provide the environment you discovered this bug in (run `ng version`)

```true
Angular CLI: 17.2.1
Node: 18.19.0
Package Manager: npm 10.2.3
OS: linux x64

Angular: 17.2.4
... core

Package                             Version
-------------------------------------------------------------
@angular-devkit/architect           0.1702.1
@angular-devkit/build-angular       17.2.1
@angular-devkit/core                17.2.1
@angular-devkit/schematics          17.2.1
@angular/animations                 17.2.2
@angular/cli                        17.2.1
@angular/common                     17.2.2
@angular/compiler                   17.2.2
@angular/compiler-cli               17.2.2
@angular/forms                      17.2.2
@angular/platform-browser           17.2.2
@angular/platform-browser-dynamic   17.2.2
@angular/router                     17.2.2
@schematics/angular                 17.2.1
rxjs                                7.8.1
typescript                          5.3.3
zone.js                             0.14.4
```

**Comment by JoostK** [comment]
What are you trying to do? This API is only available from 17.3 onwards.

**Comment by aiuoe** [comment]
v17.2.4 includes ɵoutput fn dev preview and not work

**Comment by JeanMeche** [comment]
`output()` is only available in v17.3. 17.2 doesn't include any implementation. `ɵoutput` is a symbol with no feature behind it.
